packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -2399,7 +2403,11 @@ export class MergeTree {
 		}
 		for (let i = 0; i < block.childCount; i++) {
 			const child = block.children[i];
-			len += nodeTotalLength(this, child) ?? 0;
+			const nodeLength = nodeTotalLength(this, child);
+			if (nodeLength !== undefined) {
+				len ??= 0;
```

Comment:
I was not aware of this operator! This is why code reviews are useful for both reviewer and reviewee 😄